### ISSUE TITLE
Why is the training so long? 

### ISSUE THREAD

**Issue by yeyupiaoling** [calmisential/TensorFlow2.0_SSD#8]
The training loss function is the longest.

**Comment by calmisential** [comment]
The issue has been resolved in the latest version of the code.

**Comment by yeyupiaoling** [comment]
Thank you so much.
